src/battle_util.c

```diff
@@ -4060,6 +4060,58 @@ static inline bool32 HadMoreThanHalfHpNowDoesnt(u32 battler)
              && gBattleMons[battler].hp <= cutoff);
 }
 
+#define ANIM_STAT_STAT_HP      0
+#define ANIM_STAT_STAT_ATK     1
+#define ANIM_STAT_STAT_DEF     2
+#define ANIM_STAT_STAT_SPATK   3
+#define ANIM_STAT_STAT_SPDEF   4
+#define ANIM_STAT_STAT_SPEED   5
+#define ANIM_STAT_STAT_ACC     6
+#define ANIM_STAT_STAT_EVASION 7
+static void ChooseStatBoostAnimation(u32 battler)
+{
+    u32 stat;
+    gBattleScripting.animArg1 = 0;
```

Comment:
Move this assignment to after all new variables have been declared, that should fix agbcc.

Reply:
oh right